### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by kevgliss**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,9 +284,87 @@ Examples:
...
+| text | str | Free text field with additional information |
+
+#### Roles Schema
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| id   | str | Id of the role |
+| policyUrl | str | URL with link to role permissions |
+| roleName | str | Corresponding AWS role (if any)
+| googleGroup | str | Corresponding google group |
```
Comment (mikegrima) [inline README.md]: Should we rename `googleGroup` and `secondaryApprover` to `primaryGroup` and `secondaryGroups`, respectively?

File: README.md [hunk 2]
```diff
@@ -277,9 +284,87 @@ Examples:
...
+| status | str | One of: Created, In-progress, Ready, Deprecated, In-active, Deleted |
+| notes | list(dict) | See notes schema |
+
+
+#### Notes Schema
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| date | date | Date note was created |
+| text | str | Free text field with additional information |
```
Comment (mikegrima) [inline README.md]: Add a field for the user that made the note?

**Comment by mikegrima** [comment]
Can you rebase off of the latest master?

File: README.md (hunk 2 above)
Comment (kevgliss) [reply README.md]: We could... but I was mostly just document what is currently exists. If we want to add something to the schema we should open a separate issue.

File: README.md (hunk 1 above)
Comment (kevgliss) [reply README.md]: We can make it more generic, it will break some stuff though.

**Pull request merged**